src/mockServiceWorker.js

```diff
@@ -27,6 +27,14 @@ self.addEventListener('message', async function (event) {
   const allClientIds = allClients.map((client) => client.id)
 
   switch (event.data) {
+    case 'PING': {
+      sendToClient(client, {
+        type: 'PONG',
+        payload: new Date(),
```

Comment:
Is there any intricate reason to pass this `payload`? I wonder if this affects something 🤔

Reply:
No - it was just an idea to look like there was some type of valid communication happening. It doesn't hurt to have it, but also doesn't seem to matter if it's removed.

Reply:
Seems trivial, but we need to think about this as changes to the `mockServiceWorker` file should be minimal across library updates (they require to re-initialize the worker file). I'd remove the `payload` property unless it's absolutely necessary. If ping-ponging empty events still keeps the worker alive, I'd leave it without any additional properties.